content/academy/apify_platform/getting_the_most_of_actors_on_apify_store.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Getting the most out of your public actors on Apify Store
+description: Learn how to optimize your public actors on Apify Store and monetize them by renting your actor to other platform users.
+menuWeight: 5.3
+paths:
+- apify-platform/getting-the-most-of-actors-on-apify-store
```

Comment:
I would make the filename and path shorter, this seems really cumbersome and typo friendly. Something like `get-most-of-actors` should be enough.

Reply:
Fixed